epub33/core/index.html

```diff
@@ -973,38 +1147,90 @@ <h5>Foreign Resources</h5>
 							</li>
 						</ul>
 
-						<p class="note">This exception allows EPUB Creators to include resources in the <a>EPUB
-								Container</a> that are not for use by EPUB Reading Systems. The primary case for this
-							exception is to allow data files to travel with an EPUB Publication, whether for scripts to
-							use in their constituent EPUB Content Documents or for external applications to use (e.g., a
-							scientific journal might include a data set with instructions on how to extract it from the
-							EPUB Container).</p>
+						<p>This exception allows EPUB Creators to include resources in the <a>EPUB Container</a> that
+							are not for use by EPUB Reading Systems. The primary case for this exception is to allow
+							data files to travel with an EPUB Publication, whether for scripts to use in their
+							constituent EPUB Content Documents or for external applications to use (e.g., a scientific
+							journal might include a data set with instructions on how to extract it from the EPUB
+							Container).</p>
+
+						<p>It also EPUB Creators to use Foreign Resources in Foreign Content Documents without Reading
+							Systems or <a>EPUB Conformance Checkers</a> having to understand the fallback capabilities
+							of those resources (i.e., the requirement for a fallback for the Foreign Content Document
+							covers any rendering issues).</p>
+
+						<p>This specification also exempts some elements from fallback requirements. In these cases,
+							Foreign Resources MAY be referenced without a fallback. For more information, refer to <a
+								href="#sec-xhtml-fallbacks"></a>.</p>
```

Comment:
I am a bit bothered by the separate §3.1.4.4 on Foreign Resource Restrictions. 

First of all, there are redundancies: the first paragraph of §3.1.4.4. seems to be a duplication of requirements that are defined in these sections. Also, §3.1.4.4 appears in a section on "HTML Deviations and Constraints", but the features described in that section are not really a constraint on HTML per se, but on how an EPUB specific feature binds to some HTML features.

Would it be feasible to move the content of 3.1.4.4. to §2.2.4 in some way or others? That would mean that all fallback-related definitions would be at the same place...

Reply:
I'm working at this, too. The part about exempt elements can fit in the new 2.2.3.1, but squashed in the middle is a set of requirements for the img element that have nothing to do with exemptions from fallbacks.

I'm still trying to find someplace to put that text. It may end up as a subsequent subsection to 2.2.3.1, but it's a weird mix of using CMTs and foreign resources.